lwallet/Main.hs

```diff
@@ -469,7 +469,7 @@ main = do
                 -- Serve webPort webDaedalusDbPath -> worker walletServerOuts $ \sendActions ->
                 --     walletServeWebLite sendActions webDaedalusDbPath False webPort
 
-        case CLI.sscAlgo woCommonArgs of
+        case woSscAlgo of
```

Comment:
Light wallet supports only GodTossing too, so this option can be removed as well: https://github.com/input-output-hk/cardano-sl/blob/ba4862854e4a829a129f9b8745c167de513446fa/lwallet/Main.hs#L478